### PR TITLE
[refs #00335] Update Release Documentation

### PR DESCRIPTION
**Pull request by joebell93**
## Description

* Beg for any help to automate this process.
* Amend the order of `lerna bootstrap`.
* Suggested commit message for CHANGELOG changes.

## Related Issue

#335 

## Motivation and Context

Encourage maintainer autonomny

## How Has This Been Tested?

N/A

## Markup

N/A

## Screenshots

N/A

## Types of Changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Browser Support

N/A

## Checklist

- [x] All **design and code** conforms to the [Design Contribution Guidelines](https://github.com/sky-uk/toolkit/blob/develop/CONTRIBUTING.md#design-contributions).
- [x] New functions and mixins have appropriate tests.
- [x] All new and existing tests passed.
- [x] I have added instructions on how to test my changes.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -302,25 +302,29 @@ the core maintainers or [Tom Davidson](@tom-davidson).
...
-7. Commit and push the `CHANGELOG.md` changes to `master`.
-8. Run `npm run dist` to generate `toolkit.min.css` and `toolkit-core.min.css`.
-    * Upload these to S3 (see maintainers Slack channel for bucket details):
-        1. Upload both to `sky.com/assets/[toolkit|toolkit-core]/v[version]`
-        2. Override both `/latest` 
+8. Commit and push the `CHANGELOG.md` changes to `master`.
+   * Don't worry about issue references here, a simple `"Update CHANGELOG"` or `"Update CHANGELOGs"` will do.
+9. Run `npm run dist` to generate `toolkit.min.css` and `toolkit-core.min.css`.
+    * Upload the assets of the updated packages to S3 (see maintainers Slack channel for bucket details):
+        1. Upload to `sky.com/assets/[toolkit|toolkit-core]/v[version]`.
```
Comment (steveduffin) [inline CONTRIBUTING.md]: You may need to scroll inside the modal to see Metadata fields (caught me out!)

**Comment by joebell93** [comment]
Updated @steveduffin 💯

**Comment by steveduffin** [comment]
Followed and the world didn't end 👍

**Comment by joebell93** [comment]
As this doesn't need a "release", we'll keep it in `develop` and merge into `master` with the next release

**Pull request merged**